src/main/java/it/gov/pagopa/payment/notices/service/controller/NoticeTemplatesController.java

```diff
@@ -0,0 +1,103 @@
+import it.gov.pagopa.payment.notices.service.model.ProblemJson;
+import it.gov.pagopa.payment.notices.service.service.NoticeGenerationService;
+import it.gov.pagopa.payment.notices.service.service.NoticeTemplateService;
+import it.gov.pagopa.payment.notices.service.util.OpenApiTableMetadata;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.io.FileUtils;
+import org.springframework.core.io.ByteArrayResource;
+import org.springframework.core.io.Resource;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import java.io.BufferedInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+
+/**
+ * Rest Controller containing APIs for generation request management
+ */
+@RestController
+@RequestMapping(value = "/notices/templates", produces = MediaType.APPLICATION_JSON_VALUE)
+@Validated
+@Slf4j
+@Tag(name = "Notice Templates APIs")
+public class NoticeTemplatesController {
+
+    private final NoticeTemplateService noticeTemplateService;
+
+    public NoticeTemplatesController(NoticeTemplateService noticeTemplateService) {
+        this.noticeTemplateService = noticeTemplateService;
+    }
+
+    /**
+     *
+     * @param templateId
+     * @return
+     */
+    @Operation(summary = "getTemplate",
+            description = "Return templates",
+            security = {@SecurityRequirement(name = "ApiKey")})
+    @OpenApiTableMetadata(readWriteIntense = OpenApiTableMetadata.ReadWrite.READ,
+            cacheable = true, external = true, internal = false)
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "Return template data",
+                    content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE
+                    )),
+            @ApiResponse(responseCode = "400", description = "Bad Request",
+                    content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE,
+                            schema = @Schema(implementation = ProblemJson.class))),
+            @ApiResponse(responseCode = "401",
+                    description = "Unauthorized", content = @Content(schema = @Schema())),
+            @ApiResponse(responseCode = "403",
+                    description = "Forbidden", content = @Content(schema = @Schema())),
+            @ApiResponse(responseCode = "404", description = "Folder not found or unavailable for the requirer",
+                    content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE,
+                            schema = @Schema(implementation = ProblemJson.class))),
+            @ApiResponse(responseCode = "429",
+                    description = "Too many requests", content = @Content(schema = @Schema())),
+            @ApiResponse(responseCode = "500",
```

Comment:
add also `503` to be coherent with[ this app error](https://github.com/pagopa/pagopa-print-payment-notice-service/pull/2/files#diff-32f4692d0a7844566c289ac705e5d24a0dc0d53b159d50fe19bebeb89f16945cR17)